server/src/main/scala/de/frosner/broccoli/templates/TemplateRenderer.scala

```diff
@@ -22,6 +36,20 @@ class TemplateRenderer(defaultType: ParameterType) {
       case ParameterType.Raw => value
       case ParameterType.String =>
         StringEscapeUtils.escapeJson(value)
+      case ParameterType.Integer =>
```

Comment:
Shouldn't sanitization now work depending on the value? Why is the value string? It should be a parameter value, or?

Reply:
Yes, this function will be removed. Since the tests used it I left it for now. But will clean everything up tomorrow.